_data/augene/augene.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Augene John Pak
+institution: Kamiak High School
+profile_pic: augene.jpg
+quote: "Hello."
```

Comment:
Please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the quotes to maintain the same format

Reply:
Done!